README.org

```diff
@@ -50,3 +50,39 @@ installed using =brew=. The same works with =cask= and =brewfile=. For example:
 
 - cask: [colorpicker, vlc]
 #+END_SRC
+
+If you plan on having multiple instances of the directives in this package, you should consider
+using defaults to disable auto_bootstrap (installation of brew and cask on every command), and 
+installing first with the installBrew: directive
+
+#+BEGIN_SRC yaml
+- defaults:
+    - brewfile:
+        - auto_bootstrap: false
+    - brew:
+        - auto_bootstrap: false
+    - cask:
+        - auto_bootstrap: false
+
+- installBrew:
+    - brew
+    - cask
+
+- brewfile:
+    - Brewfile
+    - brew/Brewfile
+
+- tap:
+    - caskroom/fonts
+
+- brew:
+    - git
+    - git-lfs
+    - emacs --with-cocoa --with-gnutls --with-librsvg --with-imagemagick --HEAD --use-git-head
+
+- brew: [gnupg, gnupg2]
+
+- cask: [colorpicker, vlc]
+#+END_SRC
+
+Defaults also can control stdin, stdout and stderr (disabling with false, enabling with true)
```

Comment:
Control over verbosity is very cool thing! :100: I think it's something worth having! I wonder of `dotbot` provides a mechanism for this. I would rather do `./install --verbose` than configure verbosity of `dotobot-brew` via its directives.

Reply:
Another thought. I don't remember how `dotbot` directives work. But I think that there might be name clashing. `default` is too broad and generic, unlike `brew` and `cask` it has nothing to do with `brew`.

Reply:
Defaults is the meta directive that dotbot has for this kind of configuration, it is read separately in ... context.py I want to say? and it's used precisely for this purpose in link: and shell: directives. 

I also reused the stdin/out/err control scheme that shell: uses. --verbose is used for showing log.debug() statements, not for enabling/disabling the standard input/output in subprocesses.

Reply:
> Defaults is the meta directive that dotbot has for this kind of configuration, it is read separately in ... context.py I want to say? and it's used precisely for this purpose in link: and shell: directives.

Nice. Didn't know. Could you please post some links to examples? 

> I also reused the stdin/out/err control scheme that shell: uses. --verbose is used for showing log.debug() statements, not for enabling/disabling the standard input/output in subprocesses.

Ah, I see. I wonder if we have an access to this flag. And if `--verbose` is passed, display `stdout` and `stderr`. Do you think it's possible?

P. S. aparently I need to read `dotbot` documentation. I haven't done it since I first implemented `dotbot-brew` and it was long time ago 😸

Reply:
https://github.com/anishathalye/dotbot#defaults

[The context object is](https://github.com/anishathalye/dotbot/blob/master/dotbot/context.py) where defaults are stored, each plugin has [access to the current context through `self._context`](https://github.com/anishathalye/dotbot/blob/master/dotbot/plugin.py#L10) and here's an example of [how the shell directive uses defaults](https://github.com/anishathalye/dotbot/blob/master/dotbot/plugins/shell.py#L27), notice built in directives are themselves implemented as plugins.

`--verbose` is only used for the logging and I think that is a better approach, unless we were to modify dotbot itself to have separate `--verbose` and `--loglevel` flags. Then again that would possibly be a nightmare for compatibility with other plugins, so I would not think that's a good idea.

This PR is focused on compatibility with configurations created before my changes, so nothing should change for a user (except using `brew bundle --verbose`), I may have otherwise used different default values (allow stdin/err/out by default since brew output is generally useful).